Generators/AutoNotify.cs

```diff
@@ -70,12 +70,12 @@ public void Execute(GeneratorExecutionContext context)
             // group the fields by class, and generate the source
             foreach (IGrouping<INamedTypeSymbol, IFieldSymbol> group in fieldSymbols.GroupBy(f => f.ContainingType))
             {
-                string classSource = ProcessClass(group.Key, group.ToList(), attributeSymbol, notifySymbol, context);
+                string classSource = ProcessClass(group.Key, group.ToList(), attributeSymbol, notifySymbol);
                 context.AddSource($"{group.Key.Name}_GeneratedNotify.cs", SourceText.From(classSource, Encoding.UTF8));
             }
         }
 
-        private string ProcessClass(INamedTypeSymbol classSymbol, List<IFieldSymbol> fields, ISymbol attributeSymbol, ISymbol notifySymbol, GeneratorExecutionContext context)
+        private static string ProcessClass(INamedTypeSymbol classSymbol, List<IFieldSymbol> fields, ISymbol attributeSymbol, ISymbol notifySymbol)
```

Comment:
Parameter was unused. Let me know if it's intentional in case you're going to use it in the session for some reason.